src/github.com/couchbase/sync_gateway/db/kv_change_index_writer.go

```diff
@@ -293,6 +293,7 @@ func (k *kvChangeIndexWriter) indexEntries(entries []*LogEntry, indexPartitions
 				defer entryWg.Done()
 				err := channelStorage.WriteLogEntry(logEntry)
 				if err != nil {
+					base.Warn("Error writing entry:%v", err)
```

Comment:
also, can you add a comment in either the log message or in a code comment above the log message that would explain the implications of seeing this warning?

Eg, "Error writing entry: foo.  This will be retried and should cause no harm but may have a performance penalty."